d2l-icon.html

```diff
@@ -105,7 +111,7 @@
 				this.async(function() {
 					this.__initialize();
 					this.updateStyles(); // need to re-calc iron-icon variables
-				}, 1);
```

Comment:
Is there a spot I can test it out?  I would not expect a diff between 0-1ms.  I did observe a difference between 0-100ms however.  I think the `attached` call above should address that issue.